Library/Formula/castxml.rb

```diff
@@ -0,0 +1,22 @@
+class Castxml < Formula
+  desc "C-family Abstract Syntax Tree XML Output"
+  homepage "https://github.com/CastXML/CastXML"
+  url "https://github.com/CastXML/CastXML/", :revision => "1489405912947356574477a3c10fec69e78cb6dc", :using => :git
```

Comment:
And use `.git` on the end of GitHub URLs and `head`s, rather than passing `:using => :git`, Thanks!